code-studio/scripts/build-js.js

```diff
@@ -39,6 +39,10 @@ Promise.all([
       'levels/contract_match.jsx',
       'levels/widget.js',
       'levels/external.js',
+      'levels/multi.js',
+      'levels/textMatch.js',
+      'levels/levelGroup.js',
+      'levels/dialogHelper.js',
```

Comment:
The only match I get for https://github.com/code-dot-org/code-dot-org/search?q=dialogHelper is [in `_dialog.html.haml`](https://github.com/code-dot-org/code-dot-org/blob/2b14b4b3ec3e42421db26780a357b4a682bae957/dashboard/app/views/levels/_dialog.html.haml#L139).  But I think `_dialog.html.haml` gets used lots of places?